Implement house model validations

MrFonseca opened the pull request:
- Implement validations for House;
- Set House model test;

MrFonseca committed:
Implement house model validations

- Implement validations for House;
- Set House model test;

gabrielhilal commented on app/models/house.rb:
```diff
@@ -5,6 +5,10 @@ class House < ApplicationRecord
   has_many :visits, -> { where(contract_type: nil) }
   has_many :employees, -> { where.not(contract_type: nil) }, class_name: "Visit"
 
+  validates :number, presence: true
```
tu pode combinar L8 e L9 se quiser: `validates :number, presence: true, uniqueness: { case_sensitive: false }`

MrFonseca committed:
Implement admin namespace

- Implement admin namespace;
- Implement house feature test for admin;
- Draw namespace routes;
- Install figaro gem;
- Implement houses admin views;
- Refact houses controller;

gabrielhilal reviewed (approved):
nice!
